core/src/main/scala/org/broadinstitute/dsde/rawls/jobexec/SubmissionSupervisor.scala

```diff
@@ -50,7 +50,7 @@ class SubmissionSupervisor(executionServiceCluster: ExecutionServiceCluster,
   }
 
   override val supervisorStrategy =
-    OneForOneStrategy(maxNrOfRetries = 3) {
+    OneForOneStrategy(maxNrOfRetries = -1) {
```

Comment:
i think this should just be a case of subclassing `OneForOneStrategy` and overriding `processFailure` to also log an error when it's above a given number of retries.